public/index.html

```diff
@@ -24,6 +24,15 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-QTN58X6JMS"></script>
```

Comment:
GTM is very expensive integration. But since GA lang naman.. oks lang to.. pero pag my other analytics na.. This would hurt the initial loading.. specially React, mabagal ang initial load.